src/app/components/nodes-view/nodes-view.component.ts

```diff
@@ -186,6 +187,32 @@ export class NodesViewComponent implements OnInit {
     return this.allocDataSource.data && this.allocDataSource.data.length === 0;
   }
 
+  formatColumn(){
+    if(this.nodeDataSource.data.length===0){
+      return;
+    }
+    this.nodeColumnIds.forEach((colId)=>{
+      if (colId==='indicatorIcon'){
+        return;
+      }
+
+      // Verify whether all cells in the column are empty.
+      let isEmpty:Boolean=true;
```

Comment:
Please help to change this.
```
let isEmpty:boolean = true;
```

Reply:
Thanks for your review. I have updated the code.

Reply:
I means the `Boolean` should be `boolean`

https://www.typescriptlang.org/docs/handbook/basic-types.html

Reply:
fixing this on commit.